### ISSUE TITLE
Help to setup XML output?  

### ISSUE THREAD

**Issue by Alifar** [franzip/serp-scraper#1]
Hi @franzip 

Thank you for this source code. Do you have an example code how to get the output in a XML feed? I cannot find this in the directory.

Thanks!

**Comment by franzip** [comment]
Hi @Alifar ,

what are you trying to achieve exactly? Wrapping the XML output in a feed structure?

**Comment by Alifar** [comment]
Yes, I would like to get a XML output with 50 results which holds two items (title and link) and retrieve the data via URL, like domain.com?key=KEYWORD&location=US

Can you please guide me how to accomplish this or do you have a example somewhere? I would really appreciate it!

Thanks in advance.

**Comment by franzip** [comment]
If you are trying to scrape and serialize data that do not come from any of the [search engines supported by this library](https://github.com/franzip/serp-scraper/blob/master/README.md), then you could use [thauex/simplehtmldom](https://github.com/ThauEx/SimpleHtmlDom) and [jms/serializer](https://github.com/schmittjoh/serializer) libraries.
Otherwise, you could [fork/clone the library used to serialize the data](https://github.com/franzip/serp-page-serializer) to obtain the XML structure you are looking for, as currently it is not possible to setup the XML output structure of the scraped pages programmatically.
